app/controllers/core_induction_programmes/years_controller.rb

```diff
@@ -7,7 +7,28 @@ class CoreInductionProgrammes::YearsController < ApplicationController
 
   after_action :verify_authorized
   before_action :authenticate_user!
-  before_action :load_course_year
+  before_action :load_course_year, except: %i[new create]
+
+  def new
+    authorize User
+    @core_induction_programmes = CoreInductionProgramme.all
+    @course_year = CourseYear.new
+  end
+
+  def create
+    @course_year = CourseYear.new(params.require(:course_year).permit(
+      :title, :content
```

Comment:
It would be convenient to handle title and content params for a year the same for creating and updating them - you might need to change the form in the edit view to have `model` attribute for it to work.